### ISSUE TITLE
Push it to jCenter.

### ISSUE THREAD

**Issue by vsmaks** [f2prateek/rx-receivers#6]
Hi. Thank you for your useful library. Can you push it to maven or jCenter? I having issue with:
```
compile 'com.f2prateek.rx.receivers:rx-receivers:0.1.0'

**Comment by gmarizy** [comment]
Same here

**Comment by f2prateek** [comment]
The library is already on Maven http://search.maven.org/#artifactdetails%7Ccom.f2prateek.rx.preferences%7Crx-preferences%7C1.0.0%7Caar

**Comment by gmarizy** [comment]
It's rx.preferences, not rx.receivers

**Comment by f2prateek** [comment]
Oh sorry, wrong project

**Comment by f2prateek** [comment]
You can use the snapshot releases for now:

```
repositories {
  mavenCentral()
  maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
}
```
`compile 'com.f2prateek.rx.receivers:rx-receivers:0.1.0-SNAPSHOT'`

**Comment by gmarizy** [comment]
Thank you for your fast answer !

There is a little copy/past error: https://oss.sonatype.org/content/repositories/snapshots/com/f2prateek/rx/receivers/

The artefact id is "rx-preferences" instead of "rx-receivers".

Anyway it's working with:
```
compile 'com.f2prateek.rx.receivers:rx-preferences:0.1.0-SNAPSHOT'
```

**Comment by f2prateek** [comment]
Ah thanks! Fixing in https://github.com/f2prateek/rx-receivers/pull/7

**Comment by sirvon** [comment]
this issue is still super relevant....

```
https://oss.sonatype.org/content/repositories/snapshots/com/f2prateek/rx/receivers/
```

not this..

```
maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
```

**Comment by f2prateek** [comment]
Released as 0.1.0 `compile 'com.f2prateek.rx.receivers:rx-receivers:0.1.0'`
